pandas/core/groupby.py

```diff
@@ -997,16 +997,16 @@ def tail(self, n=5):
         Examples
         --------
 
-        >>> df = DataFrame([[1, 2], [1, 4], [5, 6]],
+        >>> df = DataFrame([['a', 1], ['a', 2], ['b', 1], ['b', 2]],
                             columns=['A', 'B'])
         >>> df.groupby('A', as_index=False).tail(1)
            A  B
-        0  1  2
-        2  5  6
+        0  a  2
+        2  b  2
```

Comment:
I think the index here is not OK:

```
In [16]: df.groupby('A', as_index=False).tail(1)
Out[16]:
   A  B
1  a  2
3  b  2
```

So 1,3 instead of 0,2